doc/source/whatsnew/v1.0.0.rst

```diff
@@ -411,6 +411,24 @@ Use :meth:`arrays.IntegerArray.to_numpy` with an explicit ``na_value`` instead.
 
    a.to_numpy(dtype="float", na_value=np.nan)
 
+**value_counts returns a nullable integer dtype**
+
```

Comment:
actually, even though is a breaking change, it puts nullable in more general usage. so actually i think this is a good change.